lib/nexpose/scan_template.rb

```diff
@@ -191,82 +191,82 @@ def enable_ip_stack_fingerprinting=(enable)
       end
     end
 
-    # Enable/diisable ICMP device discovery
-    # @param [Boolean] enable or disable ICMP device discovery
-    def enable_icmp_discovery=(enable)
+    # Enable/disable ICMP device discovery
+    # @param [Boolean] enable Enable or disable ICMP device discovery
+    def enable_icmp_device_discovery=(enable)
```

Comment:
When this PR is landed, yes, we should release a new version.  This PR just makes the method names for enabling/disabling more consistent.  In this particular case, it is simply making it more inline with the `enable_(tcp|udp)_(device|service)_discovery` methods (even though the 'device discovery' part here is technically unnecessary, because there is no "ICMP service discovery")